docs/public/configuration/backend.md

```diff
@@ -319,6 +324,63 @@ Specifying a profile JSON file that contains no profile data is equivalent
 to specifying a profile JSON file containing the entire
 *Zonemaster Engine default profile*.
 
+
+## TLD URL section
+
+For the context of this section, see [TLD URL for GUI].
+
+The TLD URL section has several keys:
+* extract_url
+* maximum_lookup_time
+* suppress_log_messages
+* tld_\<TLD\> where `<TLD>` can be any string matching one of the following
+  patterns:
+  * `[a-z][a-z]+`
+  * `xn--[a-z0-9-][a-z0-9-]+`
+
+### extract_url
+
+Boolean value to enable the function described in document
+[TLD URL for GUI]. Disabling the function is the same as setting
+a global policy to block the function as described in the document.
+
+Accepted values: `yes` or `no`. Default to `yes` (enabled).
+
+### maximum_lookup_time
+
+The number of seconds each of the two lookups, DNS and RDAP, respectively, may
+maximally take. If the maximum time is reached, a timeout is executed. The
+total maximum time will be double that value.
+
+Accepted values: a positive integer. Default to 3.
+
+### suppress_log_messages
+
+Boolean value to suppress log messages in the API response to a
+[`get_tld_url` method query][API method get_tld_url].
+
+Accepted values: `yes` or `no`. Default to `no` (do not suppress).
+
+### tld_\<TLD\>
```

Comment:
This introduces a new way of encoding dictionaries. Traditionally we're using a separate section for each dictionary (`PUBLIC PROFILES` and `PRIVATE PROFILES`) and no prefixes on the keys. Here a dictionary with prefixed keys is integrated with other properties in a common section. I suggest breaking out these overrides into a section.
```
[TLD URL OVERRIDES]
xa         = [BLOCK]
xn--4cab6c = [BLOCK]
xb         = http://nic.xb/domain
example    = https://whoisweb.noc.example/domain/[DOMAIN]
```

Reply:
See updates.